test_unstructured_ingest/expected-structured-output/sharepoint/Shared Documents/ideas-page.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "type": "NarrativeText",
+    "element_id": "c08fcabe68ba13b7a7cc6592bd5513a8",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "CheckInComment": "",
+          "CheckOutType": 2,
+          "ContentTag": "{0DFE3D76-00C0-42DB-AE1B-8CF22D4B3F10},1,1",
+          "CustomizedPageStatus": 0,
+          "ETag": "\"{0DFE3D76-00C0-42DB-AE1B-8CF22D4B3F10},1\"",
+          "Exists": true,
+          "ExistsAllowThrowForPolicyFailures": true,
+          "IrmEnabled": false,
+          "Length": "6161",
+          "Level": 1,
+          "LinkingUri": null,
+          "LinkingUrl": "",
+          "MajorVersion": 1,
+          "MinorVersion": 0,
+          "Name": "ideas-page.html",
+          "ServerRelativeUrl": "/Shared Documents/ideas-page.html",
+          "TimeCreated": "2023-06-16T05:04:47Z",
+          "TimeLastModified": "2023-06-16T05:04:47Z",
+          "Title": "How to Get New Ideas",
+          "UIVersion": 512,
+          "UIVersionLabel": "1.0",
+          "UniqueId": "0dfe3d76-00c0-42db-ae1b-8cf22d4b3f10",
+          "ServerRelativePath": {
+            "DecodedUrl": "/Shared Documents/ideas-page.html"
+          }
```

Comment:
Looks like a lot of data we don't need in here. record_locator should container only data necessary to uniquely identify the document on the source system

Reply:
Seems like? If you needed to write a script to pull retrieve this file programmatically, could you do it with those pieces of information? Would you need a tenant ID? Can assume that one has credentials, but does this give you enough data to fetch that document again?

Reply:
okay, cool, let's have those three then?